gspread_pandas/client.py

```diff
@@ -384,7 +384,7 @@ def create_folder(self, path, parents=True):
         self.refresh_directories()
         return parent
 
-    def move_file(self, file_id, path, create=False):
+    def move_file(self, file_id, path, id_string, create=False):
```

Comment:
`id_string` here is the folder id, right? if so, I think `folder_id` would be more descriptive (`id_string` indicates it's an ID and a string, but it's not clear an id for what).

Since `folder_id` becomes an alternative to `path`, we should make both parameters optional with a default of `None` and a check that at least one of them is provided.